config/routes.rb

```diff
@@ -28,6 +28,8 @@
 
     post 'signin', to: 'sessions#instructor_create'
 
+    get '/quiz/:quiz_id/group/:group_id', to: 'quizzes#group_result'
```

Comment:
Change the route to be quizzes instead of quiz @Youssef-Emad  to follow one convention for quizzes routes .